src/GitReleaseManager.Core/VcsService.cs

```diff
@@ -141,7 +142,7 @@ private async Task AddAssetsAsync(string owner, string repository, string tagNam
                     {
                         if (!File.Exists(asset))
                         {
-                            var message = string.Format("Requested asset to be uploaded doesn't exist: {0}", asset);
+                            var message = string.Format(CultureInfo.InvariantCulture, "Requested asset to be uploaded doesn't exist: {0}", asset);
```

Comment:
Same as above, if this is meant to be outputted to the user eventually, it should be `CultureInfo.CurrentUiCulture`.

```suggestion
                            var message = string.Format(CultureInfo.InvariantCulture, "The requested asset to be uploaded doesn't exist: {0}", asset);
```